src/randoop/NoExceptionCheck.java

```diff
@@ -12,9 +12,11 @@
   
   // Indicates which statement is expected to return normally. 
   private final int statementIdx;
+  private String exceptionName;
   
-  public NoExceptionCheck(int statementIdx) {
+  public NoExceptionCheck(int statementIdx, String exceptionName) {
```

Comment:
This comment doesn't seem to have been addressed.
If this check indicates that no exception should be thrown, then why is one exception name being specially used?